packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import * as KeyCode from 'keycode-js';
+import WorklistDataGridPropTypes from './proptypes/WorklistDataGridPropTypes';
+import styles from './WorklistDataGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String representing an attribute which identifies the element (or elements) that labels the grid.
+   */
+  ariaLabelledby: PropTypes.string,
+  /**
+   * String that labels the grid for accessibility.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * String that will be used to identify the Grid. This value will be used as the id attribute of the overall grid container,
+   * and it will be used to prefix other id attributes used for internal componentry.
+   */
+  id: PropTypes.string.isRequired,
+  /**
+   * Data for column headers. Columns will be presented in the order given.
+   */
+  columnHeaders: WorklistDataGridPropTypes.columnHeaderShape,
+  /**
+   * Data for content in the body of the Grid. Rows will be rendered in the order given.
+   */
+  rows: PropTypes.arrayOf(WorklistDataGridPropTypes.rowShape),
+  /**
+   * String that specifies the default column width. Any valid CSS width value is accepted.
+   * This value will be used if no overriding width value is provided on a per-column basis.
+   */
+  columnHeaderDefaultWidth: PropTypes.string,
+  /**
+   * String that specifies the default row height. Any valid CSS width value is accepted.
+   * This value will be used if no overriding height value is provided on a per-row basis.
+   */
+  rowDefaultHeight: PropTypes.string,
+  /**
+   * String that specifies the column height. Any valid CSS height value is accepted.
+   */
+  columnHeaderHeight: PropTypes.string,
+  /**
+   * Number indicating the index of the column that represents row header. Index is 0 based and cannot exceed one less than the number of columns in the grid.
+   */
+  rowHeaderIndex: PropTypes.number,
```

Comment:
The Worklist Framework for MPages and Orion do not use the row header concept.  Talking with Andy Plumb, it appears that this concept is for compatibility with the flowsheet.  I totally understand that need.  However, how do we remove the row header for the Worklist in this implementation?

Reply:
@chrismichalewicz & @eawww - please provide your feedback on the following questions:
Do we support row header concept in the worklist data grid?
1. If yes, should the consumer be able to build a grid without the row header? Based on current story requirements, if a row header position is not provided, the worklist data grid treats the 1st column as the row header.
2. If yes, is there a use case for the row header to be anything other than the 1st column?

Reply:
Based on the meeting yesterday, this column index will not actually look like a header but will be used for the voice over capabilities.